h/api/test/presenters_test.py

```diff
@@ -29,6 +31,40 @@ def test_constructor_args(self):
         assert presenter.request == request
         assert presenter.annotation == annotation
 
+    def test_created_none_if_missing(self):
```

Comment:
I think `test_created_returns_None_if_missing()` would be clearer (more explicit) and consistent with some of the other test names here (e.g. it's `test_created_uses_iso_format()` not `test_created_iso_format()`), although the test naming style in this file is a mish mash.